src/testing/TestGame.java

```diff
@@ -1,1017 +0,0 @@
-package testing;
```

Comment:
split into multiple files, tests are not fundimentally changed. Moved into files (testEndGame, TestSaveLoad, TestUndo, TestMove, TestPull, TestPush)